k-distribution/tests/regression-new/let-test/test.k

```diff
@@ -0,0 +1,6 @@
+module TEST
+  imports INT
+
+  rule 0 => #let FOO = 1 #in FOO
```

Comment:
Let's add a few more rules, specifically one with 1 nested binding, one with 2. If you are happy with the output they give, then feel free to add automerge @dwightguth